pipelines/run-pipelines-on-apache-airflow/README.md

```diff
@@ -0,0 +1,332 @@
+<!--
+{% comment %}
+Copyright 2021-2021 Elyra Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+## Run pipelines on Apache Airflow
+
+A [pipeline](https://elyra.readthedocs.io/en/stable/user_guide/pipelines.html) comprises one or more nodes that are (in many cases) connected to define execution dependencies. Each node is implemented by a [component](https://elyra.readthedocs.io/en/stable/user_guide/pipeline-components.html) and typically performs only a single task, such as loading data, processing data, training a model, or sending an email. Note that in Apache Airflow components are called operators, but for the sake of consistency the Elyra documentation refers to them as components.
+
+A _generic pipeline_ comprises nodes that are implemented using _generic components_. Elyra includes generic components that run Jupyter notebooks, Python scripts, and R scripts. Generic components have in common that they are supported in every Elyra pipelines runtime environment: local/JupyterLab, Kubeflow Pipelines, and Apache Airflow.
+
+The following tutorials cover generic pipelines:
+- [Introduction to generic pipelines](../introduction-to-generic-pipelines)
+- [Run generic pipelines on Kubeflow Pipelines](../run-generic-pipelines-on-kubeflow-pipelines)
+- [Run generic pipelines on Apache Airflow](../run-generic-pipelines-on-apache-airflow)
+
+A _runtime specific_ pipeline comprises nodes that are implemented using generic components or _custom components_. Custom components are runtime specific and user-provided.
+
+In this intermediate tutorial you will learn how to add [Apache Airflow components](https://airflow.apache.org/docs/apache-airflow/stable/howto/operator/index.html) to Elyra and how to utilize them in pipelines.
+
+![The completed tutorial pipeline](doc/images/completed-tutorial-pipeline.png)
+
+The features described in this tutorial require Elyra v3.3.0. The tutorial instructions were last updated using Elyra v3.3.0 and Airflow v1.10.12.
+
+### Prerequisites
+
+- [JupyterLab 3.x with Elyra extension v3.3.0 (or newer) installed](https://elyra.readthedocs.io/en/stable/getting_started/installation.html).
```

Comment:
```suggestion
- [JupyterLab 3.x with Elyra extension v3.3 (or later) installed](https://elyra.readthedocs.io/en/stable/getting_started/installation.html).
```

P.S. I will make those updates also for the KFP tutorial, which needs to be refreshed.